### DISCUSSION TITLE
Action menu on List View and Details view not uniform

### DISCUSSION THREAD

**Discussion by mrleblanc101** [laravel/nova-issues#5177] [Q&A] [answered]
Those 2 menu should probably have a uniform styling.

![MicrosoftTeams-image (1)](https://user-images.githubusercontent.com/17392251/209002818-e80b2f19-612f-4b7c-8593-8a3c12bff7b8.png)
![MicrosoftTeams-image](https://user-images.githubusercontent.com/17392251/209002820-90eb87e8-6b44-4d68-8f5a-076baa61f406.png)

**Comment by crynobone** [comment, marked as answer]
Index resource are structurally different because it has to accommodate standalone actions and pivot actions while Detail view includes deletions and replications.

**Reply by mrleblanc101** [reply]
I don't really how this respond to my feature request. You could use the custom dropdown for both as they are functionally the same, no ?

**Reply by crynobone** [reply]
No plan for that at the moment.

**Reply by mrleblanc101** [reply]
This is why I opened it as a feature request and not an issue. 
Thanks for your time
